extensions/amp-render/1.0/component.js

```diff
@@ -34,6 +34,8 @@ const DEFAULT_GET_JSON = (url) => {
   return fetch(url).then((res) => res.json());
 };
 
+const DEFAULT_DATA = Object.create({});
```

Comment:
FYI using `Object.create({})` since `Object.create(null)` behaves in unexpected ways (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create#custom_and_null_objects)